Adyen/UI/Form/Items/Text/FormTextItemView.swift

```diff
@@ -272,7 +276,7 @@ open class FormTextItemView<ItemType: FormTextItem>: FormValidatableValueItemVie
     
     public func notifyDelegateOfMaxLengthIfNeeded() {
         let maximumLength = item.validator?.maximumLength(for: item.value) ?? .max
-        if item.value.count >= maximumLength {
+        if item.value.count == maximumLength {
```

Comment:
No chance that `item.value` will be longer? Or sanitisation will clean exceeding?

Reply:
Yeah I recall previous change had the truncation logic but the overall complexity for this feature seems high.

Reply:
This is just reverting how it was before - what it does it not great but I did not want to alter the behavior